### DISCUSSION TITLE
Create more lesson

### DISCUSSION THREAD

**Discussion by thuongtruong109** [thuongtruong1009/dev-master#12] [Q&A] [answered]
I follow your progress, this is very hard and large time:
- You can create more lessons like that such as Javascript, Boostrap, or jQuery to completely front-end knowledge
- I give you a star and fork great your repo to my wall to share them with my friend and learn more.
- Your work is very positive to the community. So, I very like your work and will support you more!
- Thank you for you!

**Comment by thuongtruong1009** [comment, marked as answer]
> I follow your progress, this is very hard and large time:
> 
> * You can create more lessons like that such as Javascript, Boostrap, or jQuery to completely front-end knowledge
> * I give you a star and fork great your repo to my wall to share them with my friend and learn more.
> * Your work is very positive to the community. So, I very like your work and will support you more!
> * Thank you for you!

Thanks for your following. I will effort to make more than great lessons so everyone can learn it!
Thank you!

**Reply by olimjon1606** [reply]
you have really good profile and thank you! for sharing your work!

**Reply by thuongtruong1009** [reply]
> you have really good profile and thank you! for sharing your work!

thank you so much ❤️ . I hope that will support for you
